doc/Makefile

```diff
@@ -30,12 +34,14 @@ help:
 
 integrate:
 	# Pull the other repositories
-	cd integration/ && \
-	git -C lxd fetch && git -C lxd reset --hard origin/main || git clone https://github.com/canonical/lxd
-	cd integration/ && \
-	git -C microceph fetch && git -C microceph reset --hard origin/main || git clone https://github.com/canonical/microceph
-	cd integration/ && \
-	git -C microovn fetch && git -C microovn reset --hard origin/main || git clone https://github.com/canonical/microovn
+	cd integration/ && ( git -C lxd fetch || git clone https://github.com/canonical/lxd )
+	git -C integration/lxd reset --hard $(LXDVERSION)
```

Comment:
Not a blocker as the outcome is the same but I am wondering if we should use `git checkout` instead of `git reset --hard` which alters the history of this local copy.

It gives the impression you are trying to reset something whereas you only want to track the branch/commit specified in the env variables:

`git -C integration/lxd checkout $(LXDVERSION)`

Reply:
This works when cloning the repo, and makes perfect sense there.
But if we have the repos cloned already and only want to update them, it might fail (because we change some files in the repos - the header and the config files - and if anything changes upstream as well, the checkout will fail).

Open for any suggestions to do this in a cleaner way!

Reply:
Ah ok, I was missing the fact that the repo is already there and has received some changes. All fine from my side then.